CMakeModules/EosioTester.cmake.in

```diff
@@ -35,14 +35,13 @@ else ( APPLE )
    set( CMAKE_CXX_FLAGS "${CMAKE_C_FLAGS} ${CMAKE_CXX_FLAGS} -Wall")
 endif ( APPLE )
 
+set( Boost_USE_MULTITHREADED      ON )
 set( Boost_USE_STATIC_LIBS ON CACHE STRING "ON or OFF" )
-find_package(Boost @Boost_MAJOR_VERSION@.@Boost_MINOR_VERSION@ EXACT REQUIRED COMPONENTS
-   date_time
-   filesystem
-   system
-   chrono
-   iostreams
-   unit_test_framework)
+IF(EXISTS @CMAKE_SOURCE_DIR@/libraries/boost)
+    add_subdirectory( @CMAKE_SOURCE_DIR@/libraries/boost ${PROJECT_BINARY_DIR}/libraries/boost EXCLUDE_FROM_ALL)
+else()
+    add_subdirectory( @CMAKE_INSTALL_FULL_DATAROOTDIR@/leap_boost ${PROJECT_BINARY_DIR}/libraries/boost EXCLUDE_FROM_ALL)
+endif()
```

Comment:
Why does the if/else exist here? It seems like we would always want the `else` branch, otherwise if someone installs multiple versions from the same build path there could be a situation where the wrong version is used.

Reply:
It depends on whether we use libtester from a build tree or an install. If from a build tree `@CMAKE_SOURCE_DIR@/libraries/boost` is present but not the other one. The opposite is true when using a leap-install.

Reply:
That's what the two different files are for though: `EosioTester.cmake.in` is used for `dev-install` & `leap-dev.deb`, while `EosioTesterBuild.cmake.in` is used for build tree.

Reply:
Oh, I had not realized that. Let me update this.